README.md

```diff
@@ -202,3 +203,9 @@ npm run test:system
 <br/>
 
 Don't see what you're looking for? Browse questions from the community or ask your own in the [Q&A section](https://github.com/zowe/zowe-cli/discussions/categories/q-a) of our repo.
+
+## **Project Structure and Governance**
+
+Zowe CLI is a component of the Zowe Open Mainframe Project, part of the Linux Foundation.
+
+To learn more about how Zowe is structured and governed, see the [Technical Steering Committee Strucutre and Governance documentation](https://github.com/zowe/community/blob/master/Technical-Steering-Committee/tsc-governance.md).
```

Comment:
Minor typo on 'Strucutre'.

Reply:
This is on our current master branch. Would it be preferred to update this here and open a new PR on master, or open the PR on master and fix this at a later sync PR on next?

Reply:
I did not realize that it already existed on master. I was more concerned about introducing new mistakes than I was about correcting old sins.

I suggest that it would be sufficient to just fix it in this 'next' PR. If someone remembers (or notices) this typo during some update to master, fix the typo as part of that PR. It seems like overkill to create a separate 'master' PR just to fix this typo.

Reply:
Sounds good, I will fix it in this PR and my open PR for deprecating getDataSet()